wiki/Performance_Troubleshooting/en.md

```diff
@@ -86,3 +86,21 @@ While this may seem like the best option (and indeed can be, if your hardware ca
 - GPUs generally aren’t meant to render 1000+ frames per second, so unexpected behaviour may occur, including lower performance or higher latency than otherwise attainable.
 
 Even so, if heat generation and potentially overworking your hardware is not considered an issue, unlimited may be the option that provides you the most stable frame times.
+
+### Custom
+
+*Note that, if you're using the Cutting Edge release stream, changing the `CustomFrameLimit` to any number above 999, will be reverted to 999; on Stable, you can set the value above 1000 with no issues.*
+*Also note that you cannot switch into this setting with the `F7` shortcut in-game, even if you had done the steps above: you'll need to switch to Custom in the Settings before quitting the game so it'll be available the next time osu! runs. Otherwise, you'll have to change the `FrameSync` manually again.*
+
+This frame limiter attempts to keep a frame rate equal to your custom preference; however, this is a setting that cannot be changed to within the in-game menu by default.
+
+To change osu!'s custom maximum framerate, you'll need to tweak some values in the config file:
+
+1. Open your osu! folder. (The default is `C:\Users\<yourPCusername>\AppData\Local\osu!` for Windows; `/Applications/osu!.app/Contents/Resources/drive_c/osu!` for Mac.)
+You can also click the "Open osu! folder" button in Options menu in-game.
+2. Close osu!.
+3. Open the `osu!.<yourPCusername>.cfg` file with Notepad or a text editor of your choice.
+4. Find the line `FrameSync = <value>` and change the value to `Custom`.
+5. Find the line `CustomFrameLimit = <value>` and change the value to your preference.
+6. Save the config file. If you haven't close osu! in the second step, please do so before saving.
+7. Re-open the game.
```

Comment:
```suggestion
1. Open your osu! folder. (The default is `C:\Users\<yourPCusername>\AppData\Local\osu!` for Windows; `/Applications/osu!.app/Contents/Resources/drive_c/osu!` for Mac.) 
  - Alternatively, you can open click the `Open current skin folder` button in Options menu and navigate up two folders.
2. Close osu!.
3. Open the `osu!.<yourPCusername>.cfg` file with Notepad or a text editor of your choice.
4. Find the line `FrameSync = <value>` and change the value to `Custom`.
5. Find the line `CustomFrameLimit = <value>` and change the value to your preference.
6. Save the config file. If you haven't close osu! in the second step, please do so before saving.
7. Re-open the game.
```

Reply:
I'm very confused at this. Aside from grammatical issues, why click the "Open current skin folder" button when the "Open osu! folder" button does exactly what you'd want?

Reply:
How did I never realize that button was there? I always just used the open current skin folder button lol

Reply:
I usually just right click osu! shortcut on desktop and select Open file location, lol.

Reply:
Thank you guys, I had changed to your 2nd suggestion.